### DISCUSSION TITLE
Permanent Password does not recognize UPPER/LOWER case if non ENG keyboard language

### DISCUSSION THREAD

**Discussion by 2play** [rustdesk/rustdesk#9342] [General]
### Bug Description

I wanted to use a password based on Hellenic (Greek) language chars.
Even though I match all needed criteria digit/upper/lower/length is does not let me save it showing UPPER/LOWER is missing

### How to Reproduce

try to create a perm password with Greek Alphabet chars
Α α, Β β, Γ γ, Δ δ, Ε ε, Ζ ζ, Η η, Θ θ, Ι ι, Κ κ, Λ λ, Μ μ, Ν ν, Ξ ξ, Ο ο, Π π, Ρ ρ, Σ σ/ς, Τ τ, Υ υ, Φ φ, Χ χ, Ψ ψ, Ω ω

### Expected Behavior

allow saving the strong password even if not ENG chars

### Operating system(s) on local side and remote side

Win10 x64

### RustDesk Version(s) on local side and remote side

1.3.0

### Screenshots

![image](https://github.com/user-attachments/assets/0dd21254-c86b-4cca-9c0b-f5928d13f36e)

**Comment by rustdesk** [comment]
@21pages

**Reply by rustdesk** [reply]
https://github.com/rustdesk/rustdesk/commit/9f9a22ec63c22e0d82418415c91a614f2f7c0235

**Comment by 2play** [comment]
Good day and than you for looking in to that in such a swift manner
I tested the nightly
https://github.com/rustdesk/rustdesk/releases/download/nightly/rustdesk-1.3.1-x86_64.exe
Just wanted to report after upgrade and testing I still get the same warning using the Hellenic Alphabet
![image](https://github.com/user-attachments/assets/08f60a7e-7a1b-4cb4-aff4-af9dbe2dc7b6)

**Reply by 21pages** [reply]
Try the latest, it's ready.

**Reply by 2play** [reply]
> Try the latest, it's ready.

Perfect! It is working flawlessly!!!
![image](https://github.com/user-attachments/assets/91fdf245-632a-47d1-876c-74d01e56575c)
